ui/src/app/components/resource-card.tsx

```diff
@@ -161,15 +161,15 @@ export class ResourceCard extends React.Component<Props, State> {
     return this.resourceType === ResourceType.DATA_SET;
   }
 
-  get actionsDisabled(): boolean {
-    return !this.writePermission;
-  }
-
-  get writePermission(): boolean {
+  get writerPermission(): boolean {
```

Comment:
So some of the boolean properties begin with `'is`. I'd follow that is/has convention and make this `hasWriterPermission` and `hasOwnerPermission`. If I just see `foo.ownerPermission()` I don't know if it has a side effect or not, or what it returns.

Reply:
In general, I think that get methods should never have side effects. Happy to have it changed, but I think following that rule is the most important thing.